skins/laika/src/components/pages/donation_form/SubmitValues.vue

```diff
@@ -4,6 +4,8 @@
 		<input type="hidden" name="zahlweise" :value="payment.type">
 		<input type="hidden" name="periode" :value="payment.interval">
 		<input type="hidden" name="betrag" :value="formattedAmount">
+		<input v-if="isBankData" type="hidden" name="iban" :value="bankdata.iban">
```

Comment:
I think you can keep the code simple and forgo the `if` and `isBankData` property. The backend will ignore the data anyways and if a hidden field is empty, it won't submit.

Reply:
I was thinking more of a weird case where a user entered valid IBAN data, then went back and changed the payment type and then submitted the form. I think in that case the IBAN data would then send along to the server if it were not for that if. Not sure if that check is really necessary though, I can go back and test it if need be.